lib/std/meta.zig

```diff
@@ -1217,10 +1217,10 @@ pub inline fn hasUniqueRepresentation(comptime T: type) bool {
 
             var sum_size = @as(usize, 0);
 
-            inline for (info.fields) |field| {
+            inline for (info.fields) |field| if (!field.is_comptime) {
                 if (!hasUniqueRepresentation(field.type)) return false;
                 sum_size += @sizeOf(field.type);
-            }
+            };
```

Comment:
```suggestion
            inline for (info.fields) |field| {
                 if (field.is_comptime) continue;
                 if (!hasUniqueRepresentation(field.type)) return false;
                 sum_size += @sizeOf(field.type);
             }
```

Reply:
I myself find the wrapping `if` easier to read, but alright, I changed it to a guard clause :)

Reply:
@nektro is not on the zig core team; you may safely ignore her code review suggestions.

![image](https://github.com/user-attachments/assets/a4e68fb8-c044-4552-b3f6-6b7b0b9c8dcf)

Only if this says "member" do you need to care about the person's opinions.

Reply:
I know, but I do usually listen to suggestions about expression nesting - I tend to write expressions as if it's Lisp, which might be harder for the average Zig programmer to follow...